README.md

```diff
@@ -54,14 +54,14 @@ You'll also need a GitHub token and a config file. (Keep reading for more info o
 1. Store your GitHub token in your system's environment by running `export MARINER_GITHUB_TOKEN={Insert your GitHub token here}`. You will either have to do this once each time you restart your system, or else configure your system to do so automatically.  
 1. Finally, run the application to find open issues in your dependencies, using the command `node index.js`.
 
-### Optional: Generating Markdown
+### Optional: Generating Markup
 
-- You can generate markdown for use in Confluence/jira
-- The generateConfluenceMarkdown() creates the markdown based on two parameters: `maxIssuesAge` and `issuesByDependency`
+- You can generate markup for use in Confluence/jira
+- The generateConfluenceMarkup() creates the markup based on two parameters: `maxIssuesAge` and `issuesByDependency`
 - `maxIssueAge` defaults to 30 days, anything over 30 days won't get written, You can edit this number.
 - Square brackets and curly braces in issue titles will be replaced by parentheses.
 - You can see an example of how to use in the `runExample.ts` file.
-Example of confluenceMarkdown.md output:
+Example of confluenceMarkup.md output:
```

Comment:
We shouldn't use the file extension to not be `.md`, since md = markdown. I think we can change this line to just "Example of confluence output:" (not mentioning any filename).

Reply:
nice catch